doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,268 @@
+      $ cd acrn-hypervisor/
+
+2. Use the following command to build ACRN for QEMU,
+
+   .. code-block:: none
+
+      make all BOARD_FILE=../acrn-hypervisor/misc/acrn-config/xmls/board-xmls/qemu.xml  SCENARIO_FILE=../acrn-hypervisor/misc/acrn-config/xmls/config-xmls/qemu/sdc.xml
+
+ For more details, refer to :ref:`getting-started-building`.
+
+3. Copy acrn.32.out from build/hypervisor to SOS guest /boot/ directory.
+
+4. Update the SOS kernel so as to include the virtio-blk driver. UOS (L2 guest) uses virtio-blk
+driver to mount rootfs.
+
+   a. Clone the repo, https://github.com/projectacrn/acrn-kernel.git.
+   b. cd acrn-kernel
+   c. Copy kernel_config_uefi_sos to .config
+   d. make olddefconfig
+   e. make menuconfig
+
+      .. figure:: images/acrn_qemu_4.png
+         :align: center
+
+   f. make
+   g. Copy bzImage go the SOS guest /boot/ directory
+
+5. Update Ubuntu GRUB to boot ACRN hypervisor and load ACRN Kernel Image. Append the following
+configuration to the :file:`/etc/grub.d/40_custom`,
+
+   .. code-block:: none
+
+      menuentry 'ACRN hypervisor' --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
+         recordfail
+         load_video
+         gfxmode $linux_gfx_mode
+         insmod gzio
+         insmod part_msdos
+         insmod ext2
+
+         echo 'Loading ACRN hypervisor with SDC scenario ...'
+         multiboot --quirk-modules-after-kernel /boot/acrn.32.out
+         module /boot/bzImage_sos Linux_bzImage
+      }
+
+6. Update grub ``sudo update-grub``.
+
+7. Shutdown the guest and relaunch using, virsh start ACRNSOS --console and select ACRN hypervisor from grub menu to launch SOS running on top of ACRN.
+This can be verified from dmesg as shown below,
+
+   .. code-block:: none
+
+      guestl1@ACRNSOS:~$ dmesg | grep ACRN
+      [    0.000000] Hypervisor detected: ACRN
+      [    2.337176] ACRNTrace: Initialized acrn trace module with 4 cpu
+      [    2.368358] ACRN HVLog: Initialized hvlog module with 4 cpu
+      [    2.727905] systemd[1]: Set hostname to <ACRNSOS>.
+
+8. When shutting down, make sure to cleanly destroy the SOS VM to prevent crashes in subsequent boots. This can be done using,
+
+      .. code-block:: none
+
+         virsh destory ACRNSOS, where ACRNSOS is the virsh domain name.
+
+
+SOS Networking updates for UOS
+******************************
+If you are planning to enable networking for UOS(L2 guest), the following steps might be helpful.
+
+1. Edit your :file:`/etc/netplan/01-netcfg.yaml` file to add acrn-br0 as below,
+
+   .. code-block:: none
+
+      network:
+         version: 2
+         renderer: networkd
+         ethernets:
+            enp1s0:
+               dhcp4: no
+         bridges:
+            acrn-br0:
+               interfaces: [enp1s0]
+               dhcp4: true
+               dhcp6: no
+
+2. Apply the new network configuration by,
+
+      a. cd /etc/netplan
```

Comment:
Probably better put in a `code-block` section

Reply:
done.